web/pages/create-dashboard.tsx

```diff
@@ -0,0 +1,94 @@
+import { useState } from 'react'
+import { Spacer } from 'web/components/layout/spacer'
+import { Page } from 'web/components/page'
+import { Title } from 'web/components/title'
+import Textarea from 'react-expanding-textarea'
+
+import { TextEditor, useTextEditor } from 'web/components/editor'
+import { createDashboard } from 'web/lib/firebase/api'
+import clsx from 'clsx'
+import { useRouter } from 'next/router'
```

Comment:
You can actually use the static router here and it's a bit simpler: 
`import Router from 'next/router'`
`Router.push(...)`